artifacts/deploy/webhook-configuration.yaml

```diff
@@ -223,6 +223,20 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: [ "v1" ]
     timeoutSeconds: 3
+  - name: federatedresourcequotalimit.karmada.io
+    rules:
+      - operations: ["CREATE", "UPDATE"]
+        apiGroups: ["flink.apache.org"]
```

Comment:
Note: This is a placeholder that I wanted some feedback from the community for. Currently if we set the admission webhook to apply to all resources, the webhook becomes pretty chatty... 

Wondering if there is any recommendation on how we could limit the webhook to only resources linked to a FederatedResourceQuota.